docs/platform/release-notes/tizen-7-0-m2.md

```diff
@@ -0,0 +1,543 @@
+      - ML_TRAIN_LAYER_TYPE_LAYER_NORMALIZATION
+      - ML_TRAIN_LAYER_TYPE_POSITIONAL_ENCODING
+    - .NET API has been added. Now, It is possible to construct, control, and train a machine learning model in Tizen device with Tizen.ManchineLearning.Trainng .NET Class.
+      - It provides :
+        - Tizen.MachineLeanring.Train.Model
+        - Tizen.MachineLearning.Train.Layer
+        - Tizen.MachineLearning.Train.Optimizer
+        - Tizen.MachineLearning.Train.Dataset
+        - And more.
+    - Training API in WebAPIs has been added for creation and training new models for machine learning on Tizen.
+      - It contains:
+        - Layer creattionLayer(LayterType type);
+        - Dataset createFileDataset()
+        - Layer createOptimizer()
+        - Model createModelWithConfiguration()
+        - Model createMode()
+        - And more.
+- NNStreamer updates
+  - NNStreamer has been upgraded to version 2.2.
+  - New tensor stream type, Float16, has been added.
+  - Edge AI functionality has been added.
+    - nnstreamer-edge package has been newly added. It provides interfaces to support data connection and AI offloading feature between edge devices.
+    - NNStreamer and related plugins now support distributed AI inference by using the interfaces for edge AI.
+  - User customized tensorflow2-lite of tensor filter has been supported. Developers may designate user-supplied tf2-lite instead of the default one.
+  - Apache MXNet filter has been newly added.
+  - NXP DeepViewRT filter has been newly added.
+  - PyTorch has been upgraded to version 1.10.2.
+  - Error messages, exception handling, and documentation have been improved for application / pipeline writers.
+  - The Rank limit of tensor stream has been increased from 4 to 8. (Experimental)
+  - edgesrc / edgesink element newly added to support publish-subscribe communication. (Experimental)
+- NNTrainer updates
+  - New layers to support advanced training methods have been added.
+    - Attention, MoL Attention, Multi-Head Attention Layer
+    - Positional Encoding Layer
+    - Layer Normalization Layer
+  - Support for Transformer with Multi-Head Attention Layer has been added.
+  - Memory Optimization Scheme has been improved.
+    - Refactoring Memory Scheduler, Memory Pool, Tensor Pool for better Performance
+    - Consume only 1/3.9 Peak Memory of TFLite v2.9 for VGG Model with better Latency (1/1.49)
+  - Experimental Features (Turned off by Default)
+    - Support for Batch-wise Parallelization has been added.
+    - Support for Learning Rate Scheduling has been added.
+    - Support for Leaky Relu Layer has been added.
+    - Tensorflow-lite Exporter
+    - Support for Gradient Clipping by Global Norm has been added.
+    - Scheduled memory swapping to further minimize peak memory consumption, which reduces the peak down to 1/4 of the conventional nntrainer.
+- Neural Network Runtime
+  - Runtime has been supported to run nnpackage with multiple models.

```

Comment:
```suggestion
  - Runtime has been supported to run NNPackage with multiple models.

```

Reply:
`nnpackage` is a proper notation. Please refer to https://github.com/Samsung/ONE/blob/master/nnpackage/spec/10_packaging_and_manifest.md